### PR TITLE
Adds i18n to the site

### LINKED ISSUE #16: Add a translation framework to the site
Add `next-translate` as translation framework.

### PR DESCRIPTION
**Pull request by mbonfoster**
This PR adds i18n to the site, but depends on using a server since next export doesn't support i18n.
See:
https://nextjs.org/docs/advanced-features/i18n-routing#how-does-this-work-with-static-generation
https://github.com/vercel/next.js/discussions/18957
https://github.com/vercel/next.js/issues/18318

Merging this PR to master will deploy the site to ElasticBeanstalk on the bloq prod AWS account.
We should also integrate this repository with transifex.
Closes #16

### CODE REVIEW

**Review by raiseerco — DISMISSED** [review]

**Review by raiseerco — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+os: linux
+dist: focal
+
+language: node_js
+node_js:
+  - '12' # AWS default
```
Comment (gabmontes) [inline .travis.yml:6]: Let's start use Node 14 and match the `Dockerfile`.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+git:
+  submodules: false
```
Comment (gabmontes) [inline .travis.yml:9]: This is not required and can be removed.

File: scripts/1-site [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+  echo "No packages have changes, skipping deploy"
+  exit 0
+fi
+
+echo "Cleaning up and installing production dependencies only"
+install-prod-deps-only
+
+# ------------------------------------------------------------------------------
+
+# if [ -n "$(echo "$CHANGED" | { grep pure-site || true; })" ]; then
```
Comment (gabmontes) [inline scripts/1-site:97]: This line (and closing line at the end) should be enabled. Correct?

File: scripts/1-site [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+echo "APP WEB SERVER"
```
Comment (gabmontes) [inline scripts/1-site:99]: Could be simplified to just "WEB SERVER" here and below.

**Review by gabmontes — CHANGES_REQUESTED** [review]
Only minor comments. In general looks very good!!

**Review by gabmontes — APPROVED** [review]

**Review by raiseerco — APPROVED** [review]

**Pull request merged**